### PR TITLE
plugins: LFGauss: initialize output

### PR DESCRIPTION
**Pull request by snappizz**
tested and seems to be working

### CODE REVIEW

File: server/plugins/LFUGens.cpp [hunk 1]
```diff
@@ -325,13 +325,13 @@ void Vibrato_next(Vibrato *unit, int inNumSamples)
 {
 	float *out = ZOUT(0);
 	float *in = ZIN(0);
-	
+
```
Comment (danstowell) [inline server/plugins/LFUGens.cpp]: these whitespace changes are superfluous to the change. are you aware of them?

File: server/plugins/LFUGens.cpp [hunk 2]
```diff
@@ -897,7 +897,9 @@ void LFGauss_Ctor(LFGauss* unit)
 	}
 	unit->mPhase = -1.0;
 
-	//LFGauss_next_k(unit, 1);
```
Comment (danstowell) [inline server/plugins/LFUGens.cpp]: hmmm so it seems there _was_ an init call in olden times, but it was commented out. any idea why?

File: server/plugins/LFUGens.cpp (hunk 1 above)
Comment (snappizz) [inline server/plugins/LFUGens.cpp]: My bad. I generally run a git diff before pushing but sometimes I forget.

File: server/plugins/LFUGens.cpp (hunk 2 above)
Comment (snappizz) [inline server/plugins/LFUGens.cpp]: Only @telephon would know why, if he remembers. It was introduced in the initial commit for LFGauss in 9bfb7474.

I could have simply uncommented it, but that would be a behavior change.

Comment (danstowell) [inline server/plugins/LFUGens.cpp]: Thanks, I see. No objections from me

Comment (telephon) [inline server/plugins/LFUGens.cpp]: I don't remember why I commented it out. Maybe because it seemed unnecessary at the time, but I'm not sure about that.

**Comment by snappizz** [comment]
ping

**Pull request merged**